Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDelegate.swift

```diff
@@ -255,5 +255,25 @@ extension FunctionalTableData {
 				cellConfig.actions.contextMenuConfiguration?.previewContentCommitter?(animator.previewViewController)
 			}
 		}
+		
+		/// Requires iOS 13.0+
+		/// Whether multi selection is available when editing mode is enabled.
+		/// Based on the tableView's `allowsMultipleSelectionDuringEditing` property.
+		/// https://developer.apple.com/documentation/uikit/uitableviewdelegate/selecting_multiple_items_with_a_two-finger_pan_gesture
+		@available(iOS 13.0, *)
+		public func tableView(_ tableView: UITableView, shouldBeginMultipleSelectionInteractionAt indexPath: IndexPath) -> Bool {
+			return tableView.allowsMultipleSelectionDuringEditing
```

Comment:
Any reason this check couldn't include the index paths as well to control which cells trigger multi select? 

```swift
public func tableView(_ tableView: UITableView, shouldBeginMultipleSelectionInteractionAt indexPath: IndexPath) -> Bool {
	let cellConfig = data.sections[indexPath]
	let shouldBeginMultiSelect = cellConfig?.actions.shouldBeginMultiSelectAction?() ?? false
	return tableView.allowsMultipleSelectionDuringEditing && shouldBeginMultiSelect
}
```

Reply:
Could do that 👍  Not sure there's a use case where you'd only want multi select available from certain cells, but no harm in supporting it